transcom/mymove
# INT B-21490 Office users can switch to their secondary duty locations (Review after B-21492)

## [B-21490](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21490)

## Summary

Office users are able to switch between their primary and secondary transportation office GBLOCs similar to an HQ user.

The bulk of the secondary transportation office changes were added in B-21492 and are also included in this PR.

Also includes G-Unit's B-21627 hotfix

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the admin app
2. Create several office users of various roles each with two transportation offices. At least one should be a Services Counseling **Supervisor**.
3. Log in as each of the users, toggle between your GBLOCs with the dropdown in the top right, switch between queues, export your queue, just try to break it.
4. As your SC Supervisor verify the origin location dropdown filter's options update with the selected GBLOC.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
